src/js/containers/agencyV2/visualizations/ObligationsByAwardTypeContainer.jsx

```diff
@@ -43,79 +44,85 @@ export default function ObligationsByAwardTypeContainer({ fiscalYear, windowWidt
         };
```

Comment:
Can we add a conditional here to update noData to false.

Reply:
done